Failure looks related
```
======================================================================
FAIL: corehq.apps.hqadmin.tests.test_views:AdminRestoreViewTests.test_get_context_data
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/hqadmin/tests/test_views.py", line 39, in test_get_context_data
    'report_row_counts': {},
AssertionError: {'hide_xml': False, 'location_type_counts': {}, 'restore_id': None, 'status_code [truncated]... != {'hide_xml': False, 'restore_id': None, 'status_code': 500, 'num_locations': 0,  [truncated]...
  {'case_type_counts': {},
   'foo': 'bar',
   'hide_xml': False,
   'location_type_counts': {},
   'num_cases': 0,
-  'num_ledger_entries': 0,
   'num_locations': 0,
   'num_reports': 0,
   'payload': '<error>Unexpected restore response 500: bad response. If you believe this is a bug please report an issue.</error>\n',
   'report_row_counts': {},
   'restore_id': None,
   'status_code': 500,
   'timing_data': [],
   'view': 'AdminRestoreView'}
----------------------------------------------------------------------
Ran 382 tests in 26.647s
FAILED (SKIP=119, failures=1)
```

--- a/corehq/apps/hqadmin/tests/test_views.py
+++ b/corehq/apps/hqadmin/tests/test_views.py
@@ -37,4 +37,5 @@ class AdminRestoreViewTests(SimpleTestCase):
                 'case_type_counts': {},
                 'location_type_counts': {},
                 'report_row_counts': {},
+                'num_ledger_entries': 0,
             })

